txnkv/transaction/txn.go

```diff
@@ -594,6 +650,185 @@ func (txn *KVTxn) LockKeysWithWaitTime(ctx context.Context, lockWaitTime int64,
 	return txn.LockKeys(ctx, lockCtx, keysInput...)
 }
 
+// StartAggressiveLocking makes the transaction enters aggressive locking state.
```

Comment:
The aggressive locked keys processing functions are complex, we may need more SQL tests to verify it's working as expected.